src/store/plugins/tipUrl.js

```diff
@@ -11,8 +11,7 @@ export default (store) => store.registerModule('tipUrl', {
     status: ({ verifiedUrls, blacklistedUrls }) => (tipUrl) => {
       store.dispatch('tipUrl/ensureFetched');
       if (!tipUrl) return 'default';
-      const twitterProfile = getTwitterAccountUrl(tipUrl);
-      const url = twitterProfile || tipUrl;
+      const url = tipUrl;
```

Comment:
Why did you remove the `getTwitterAccountUrl` usage from here? It is not related to this task.